pkg/login/social/connectors/social_base.go

```diff
@@ -268,5 +278,16 @@ func validateInfo(info *social.OAuthInfo, oldInfo *social.OAuthInfo, requester i
 		validation.AllowAssignGrafanaAdminValidator(info, oldInfo, requester),
 		validation.SkipOrgRoleSyncAllowAssignGrafanaAdminValidator,
 		validation.OrgAttributePathValidator(info, oldInfo, requester),
-		validation.OrgMappingValidator(info, oldInfo, requester))
+		validation.OrgMappingValidator(info, oldInfo, requester),
+		validateLoginPrompt,
+	)
+}
+
+func validateLoginPrompt(info *social.OAuthInfo, requester identity.Requester) error {
+	prompt := info.LoginPrompt
+
+	if prompt != "" && prompt != "login" && prompt != "consent" && prompt != "select_account" {
```

Comment:
nit: it might be worth having constants for `login`, `consent` and `select_account`.